### DISCUSSION TITLE
Any plan to support Backlink or/and Graph view?

### DISCUSSION THREAD

**Discussion by yrong** [requarks/wiki#3383] [Help / Questions]
**Question**

Any plan to support Backlink or/and Graph view like [these products](https://www.notion.so/Artificial-Brain-Networked-notebook-a131b468fc6f43218fb8105430304709) ?

**Comment by Smankusors** [comment]
this should be feature request instead

**Comment by yrong** [comment]
after some time play with wiki code base seems backlink is doable from data model layer perspective . The pull request as following has nothing to do with UI side and is more like a experiment than a formal feature commit. 

https://github.com/Requarks/wiki/pull/2023

Hope we can improve back link as first class citizen from all perspective(like tag)
